test/error/memoize_output_invalid.cpp

```diff
@@ -0,0 +1,12 @@
+#include <Halide.h>
+using namespace Halide;
+
+int main(int argc, char **argv) {
+    Var x{"x"};
+    Func f{"f"};
+    f(x) = 0.0f;
+    f(x) += 1;
+    f.memoize();
+
```

Comment:
error tests seem to all end with printf("Success!\n"); I'm not sure if that's necessary for the build system to treat them as successfully failed (e.g. does it just check that success wasn't printed?).

Reply:
Printing `Success!` isn't necessary. It checks the exit code, too.

Reply:
Ah, but we want the test to fail if it gets there... ugh.